recipes/R/Retail - GradientBoosting/R/applicationScorer.R

```diff
@@ -52,13 +52,14 @@ applicationScorer <- setRefClass("applicationScorer",
                                                             user_token = configurationJSON$ML_FRAMEWORK_IMS_TOKEN, 
                                                             service_token = configurationJSON$ML_FRAMEWORK_IMS_ML_TOKEN)
 
-      df <- reader$load(configurationJSON$scoringDataSetId, configurationJSON$ML_FRAMEWORK_IMS_ORG_ID)
+      df <- reader$load(configurationJSON$dataSetId, configurationJSON$ML_FRAMEWORK_IMS_ORG_ID, batch_id = configurationJSON$batchId)
```

Comment:
Same comment as above

Reply:
@sathyarun18
If we do not change it, UI will not map specified dataSetId into trainingDataSetId as expected, it will always use some random dataset id. Not sure why, but I have tested it and that is the thing here. With this changed name for dataSetId, the same will be taken for training. You can check without changing the name on this link : https://platform.adobe.com/studio/instances/47b459cc-43cd-4e98-89ce-9f280cd80536

You can see what dataset id I specify and, from the logs, what dataset id is taken.